src/2023/index.html

```diff
@@ -178,6 +178,20 @@ <h2 class="eyebrow">Sponsors</h2>
4,325.21h7.43C333.44,325.21,350.58,313.47,350.58,289.51ZM316,304.6H306.6l2.64-20.13h8.27c6.47,0,11.14,2.76,11.14,8.75C328.65,301,322.9,304.6,316,304.6Z" transform="translate(-10.42 -11.42)"/><path class="cls-2" d="M299.54,153.59c-3.4-17.83-12-33.7-23.72-43.55l-0.3-.25-0.15-.16a31.31,31.31,0,0,0-3.77-2.38,31,31,0,0,1-14,14.32c-0.5-8.07-8.47-18.29-14.42-24.87-3.94-.9-7.85-1.65-11.66-2.29,1.28,22.81-12.73,39.53-12.73,39.53-10.1-18-49.06-33.18-64.57-38.62-6.15,1.28-11.84,2.78-17.09,4.41,8.22,13.91,5.62,28.88,5.62,28.88-7.34-4.36-20-4.86-32-4a100.22,100.22,0,0,0-2.57,93.65c12.79,4.38,29.49,6.66,49.39,6.66,22.27,0,43.22-9,57.5-24.68l5.7-6.27,4.65,7.14c9.71,14.91,23.45,23.81,36.76,23.81h0a30.63,30.63,0,0,0,6.17-.63C292.07,219.36,306,187.67,299.54,153.59Zm-141,46.59a5.81,5.81,0,0,1-6.28-3.29,3.62,3.62,0,0,1,2.8-5.1,5.82,5.82,0,0,1,6.28,3.29A3.62,3.62,0,0,1,158.5,200.18Zm-2.87-51.27a11.42,11.42,0,0,1,1.95-6.39l20.78,4.13A11.47,11.47,0,1,1,155.63,148.91Zm22,68.46a5.83,5.83,0,0,1-6.28-3.29,3.61,3.61,0,0,1,2.8-5.1,5.81,5.81,0,0,1,6.28,3.29A3.62,3.62,0,0,1,177.61,217.37Zm4.09-21.84a5.82,5.82,0,0,1-6.28-3.29,3.62,3.62,0,0,1,2.8-5.1,5.81,5.81,0,0,1,6.28,3.29A3.61,3.61,0,0,1,181.7,195.53Zm62.15-3a2.89,2.89,0,0,1-2.24-4.08,4.66,4.66,0,0,1,5-2.63,2.89,2.89,0,0,1,2.24,4.08A4.65,4.65,0,0,1,243.85,192.58Zm8.3,14.84a4.65,4.65,0,0,1-5,2.64,2.89,2.89,0,0,1-2.24-4.08,4.66,4.66,0,0,1,5-2.64A2.9,2.9,0,0,1,252.15,207.41Zm0-50.76a8.6,8.6,0,0,1-8.44-10.3l15.58-3.1A8.61,8.61,0,0,1,252.17,156.65Zm15.26,37a4.65,4.65,0,0,1-5,2.63,2.89,2.89,0,0,1-2.24-4.08,4.66,4.66,0,0,1,5-2.63A2.9,2.9,0,0,1,267.43,193.67Z" transform="translate(-10.42 -11.42)"/><path class="cls-3" d="M216.65,207.41c-11.42,12.06-27.34,21.08-45.59,24.87,9.17,5,22.5,8.13,37.36,8.13,15.31,0,29-3.34,38.19-8.59C235.19,227.85,224.66,219.23,216.65,207.41Z" transform="translate(-10.42 -11.42)"/><path class="cls-3" d="M110.81,124.57a214.39,214.39,0,0,0-24.54,3.26C73.81,139.05,67,151.73,67,163.72c0,16.32,10.71,35.94,25.48,46.65a59.8,59.8,0,0,0,15.79,7.84A100.22,100.22,0,0,1,110.81,124.57Z" transform="translate(-10.42 -11.42)"/><path class="cls-4" d="M235.14,175.06a2.55,2.55,0,0,0-.29-0.69c-0.61-1.34-2.69-1.49-5.2-1.67-2.32-.17-5.4-0.26-8.66-0.26s-6.34.09-8.66,0.26c-2.65.19-4.81,0.34-5.29,1.89a2.06,2.06,0,0,0-.11.35c-0.29,1.31.45,2.1,2.87,4.71,1.26,1.35,2.82,3,4.26,4.82s2.62,3.54,3.44,4.77c1.35,2,2,3,3.42,3h0.14c2.14,0,7.42-4.29,10.25-7.8C234.42,180.62,235.58,176.82,235.14,175.06Z" transform="translate(-10.42 -11.42)"/><path class="cls-5" d="M178.35,146.65a11.47,11.47,0,1,1-20.78-4.13l-27.05-5.38a33,33,0,1,0,63.3,13.16c0-.19,0-0.38,0-0.57Z" transform="translate(-10.42 -11.42)"/><path class="cls-5" d="M279.6,139.21l-20.29,4a8.61,8.61,0,1,1-15.58,3.1l2.43-.48-14,2.79c0,0.14,0,.29,0,0.43A24.76,24.76,0,1,0,279.6,139.21Z" transform="translate(-10.42 -11.42)"/><path class="cls-6" d="M169.77,144.95l-3.62-.72A4.78,4.78,0,1,0,169.77,144.95Zm83.11-.42-2.71.54A3.58,3.58,0,1,0,252.88,144.53Z" transform="translate(-10.42 -11.42)"/></svg>
```

Comment:
I try to manually optimize since we don't have an automatic image processing pipeline.

I sometimes use command line tools or these online tools that do it in browser, but still "locally":

- [Squoosh](https://squoosh.app/) for raster images
- [SVGOMG](https://jakearchibald.github.io/svgomg/) for SVG

Can you try it? We may get some savings here and they add up.

Also, I would remove the unused `id` attribute and change the `title` attribute to `GrUSP`.

Reply:
Nice, came down from 5KB to 3KB. I think it didn't impact the quality at all (?)